lib/trino-plugin-toolkit/src/test/resources/file-based-disallow-function-rule-combination-2.json

```diff
@@ -0,0 +1,16 @@
+{
+    "functions": [
+        {
+            "catalog": "sample_catalog",
+            "function_kinds": [
+                "SCALAR",
+                "AGGREGATE",
+                "TABLE"
```

Comment:
I don't see difference maker between this test and `file-based-disallow-function-rule-combination.json`. Did you want to remove `TABLE` instead of `WINDOW`? If yes, could you please rename this file `-2` -> `-without-table`?

Reply:
Yes, by mistake I remove wrong value.
Good idea with `-without-table`, renamed.